src/article-editor.jsx

```diff
@@ -66,7 +67,7 @@ var ArticleEditor = React.createClass({
 
     render: function() {
 
-        return <div className="framework-perseus perseus-article">
+        return <div id="perseus" className="framework-perseus perseus-article">
```

Comment:
I had a look at editor.less and there are other styles which depend on ```#perseus``` as well as other id based rules, e.g. ```#problemarea```, ```#workarea```, ```#answer_area```, etc.  Also, there's this comment above these styles:
```
// DOM-based styles that overwrite rules set in khan-exercise.css
```
I don't want to mess stuff up further down the pipeline so I'm going to revert this change.  I'll create an issue to clean up id related styles.